test/issues_test.js

```diff
@@ -210,5 +210,21 @@ describe('Issues', function () {
       verifyAll()
       done()
     })
+    it('responds with Bugs given jql query param', function (done) {
+      var req = {
+        query: {
+          jql: 'issueType = Bug'
+        }
+      }
+      var s = [{
+        name: 'foo'
+      }, {
+        name: 'bar'
+      }]
+      _issues.expects('filter').withArgs('issueType = Bug').yields(null, s)
```

Comment:
true... although I'm not that used to mocking methods within the class I'm testing... I'll see what is the best way to get that code covered properly...